<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Film</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>571</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajout film</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Genre&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>100</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>260</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>200</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>370</y>
          <width>221</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter et envoyer un e-mail</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>260</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Durée&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Id&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>212</x>
          <y>151</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>comédie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dramatique</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>-2</x>
          <y>-1</y>
          <width>1021</width>
          <height>491</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/new/prefix1/istockphoto-970785084-170667a.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_7</zorder>
       <zorder>label_3</zorder>
       <zorder>label_4</zorder>
       <zorder>lineEdit_id</zorder>
       <zorder>lineEdit_nom</zorder>
       <zorder>timeEdit</zorder>
       <zorder>pushButton_4</zorder>
       <zorder>dateEdit</zorder>
       <zorder>pushButton</zorder>
       <zorder>label</zorder>
       <zorder>label_5</zorder>
       <zorder>label_2</zorder>
       <zorder>comboBox</zorder>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Affichage des film</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>1081</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string>Liste des films</string>
        </property>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>120</y>
           <width>20</width>
           <height>311</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>130</y>
           <width>190</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
     border-style: outset;
     border-width: 4px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px

</string>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_supprimer">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>180</y>
           <width>190</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
     border-style: outset;
     border-width: 4px;
     border-radius: 10px;
    
border-color: rgb(255, 255, 255);
     font: bold 14px;
     min-width: 10em;
     padding: 6px</string>
         </property>
         <property name="text">
          <string>supprimer par id</string>
         </property>
        </widget>
        <widget class="QTableView" name="tablefilm">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>541</width>
           <height>311</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">  border: 2px solid grey;
border-color: rgb(0, 0, 0);
     border-radius: 5px;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_supprimer">
         <property name="geometry">
          <rect>
           <x>880</x>
           <y>180</y>
           <width>186</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">  border: 2px solid grey;
border-color: rgb(0, 0, 0);
     border-radius: 5px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>890</x>
           <y>430</y>
           <width>190</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
     border-style: outset;
     border-width: 4px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px</string>
         </property>
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>190</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
     border-style: outset;
     border-width: 4px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px</string>
         </property>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_18">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>71</y>
           <width>361</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">  border: 4px solid grey;
border-color: rgb(0, 0, 0);
     border-radius: 5px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>440</y>
           <width>541</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
  border: 4px solid grey;
border-color: rgb(0, 0, 0);
     border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>  Trier par </string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>genre</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>50</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>nom</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>440</y>
           <width>190</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
     border-style: outset;
     border-width: 4px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px</string>
         </property>
         <item>
          <property name="text">
           <string>select</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>genre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>duree</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>date_de_sortie</string>
          </property>
         </item>
        </widget>
        <widget class="QRadioButton" name="radioButton_8">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>450</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Ordre desc</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_7">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>450</y>
           <width>95</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Ordre asc</string>
         </property>
        </widget>
        <zorder>label_6</zorder>
        <zorder>tablefilm</zorder>
        <zorder>verticalScrollBar</zorder>
        <zorder>pushButton_2</zorder>
        <zorder>pushButton_supprimer</zorder>
        <zorder>lineEdit_supprimer</zorder>
        <zorder>pushButton_5</zorder>
        <zorder>pushButton_6</zorder>
        <zorder>lineEdit_18</zorder>
        <zorder>radioButton</zorder>
        <zorder>radioButton_2</zorder>
        <zorder>radioButton_3</zorder>
        <zorder>comboBox2</zorder>
        <zorder>radioButton_8</zorder>
        <zorder>radioButton_7</zorder>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>-60</x>
          <y>0</y>
          <width>1231</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/new/prefix1/istockphoto-970785084-170667a.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label_8</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>evenement</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>1061</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>ajouter even</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_nom_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>70</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Id&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>130</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Date début&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit_2">
        <property name="geometry">
         <rect>
          <x>263</x>
          <y>261</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit_1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>190</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>260</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Date fin&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_even">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>liste des even</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>921</width>
          <height>441</height>
         </rect>
        </property>
        <property name="title">
         <string>Liste des even</string>
        </property>
        <widget class="QToolButton" name="toolButton_2">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>30</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton_2">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>30</y>
           <width>187</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Recherche</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>321</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QScrollBar" name="verticalScrollBar_2">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>70</y>
           <width>20</width>
           <height>301</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QFontComboBox" name="fontComboBox_3">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>390</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="currentText">
          <string>Tirer par</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../../../../Qt/Nouveau dossier/images.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>390</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_supprimer_2">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>130</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>supprimer par id</string>
         </property>
        </widget>
        <widget class="QTableView" name="tablefilm_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>491</width>
           <height>261</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_supprimer_2">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>160</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
